templates/index.html

```diff
@@ -68,11 +68,21 @@ <h3>Status</h3>
         <span> - </span>
         <span class="roomday-comment-text">{{ room_day.comment|e }}</span>
       </span>
+      <br>
+      <button title="Click this if you are currently editing this room" type="button" class="roomday-checkin">CHECK IN</button>
```

Comment:
A fancier UI might hide the checkin button when a room is checked in by someone, and hide the checkout+currently-editing text when the room is unoccupied. ;) Just a minor feature though, not required.